### ISSUE TITLE
VWP Project Model Summaries

### ISSUE THREAD

**Issue by jdkleiner** [HARPgroup/vahydro#317]
This can serve as a place for housing ideas for the best ways to communicate modeling findings/results 
- can serve the function of relaying information to managers 
- will also serve as the items to be includes in CIA model summary permit fact sheet attachements

**Comment by ryugreen** [comment]
My quick back of the napkin thoughts for items that should be in each - aka kinda boilerplate (i know nobody likes the sound of that).

- 1-2 paragraphs Intro covering VA Hydro surface water model basics at high level (much more brief version of what is in state plan).  What are the inputs, assumptions, etc. This would be largely the same across all model summaries with tweaks over time as needed. 
- Maybe a short paragraph describing the proposed project?
- List of of scenarios run with brief description of each and identification of run-ids if not labeled in human readable language in the follow up tables.
- Glossary of properties/variables used in #2 
- Conclusion/Recommendation, paragraph or two identifying the final result from the analysis. Covering things like: Is there unmet demand as proposed? What are the impacts downstream? Reservoir drawdown if applicable. If the applicant's proposed limits and controls are not sustainable, what changes can we recommend and how does this impact the issues noted above. This paragraph would be very case by case so not really boilerplate at all but we could likely get a good basic structure down for how to approach it.

**Comment by rburghol** [comment]
Ryan - these sound wise to me.  One thing I'd assume is that the description of the proposed project would be limited, in this section, to what is their annual, monthly and daily maximums, and what flowbys they have to observe, do they have storage, i.e., how they are modeled.

**Comment by ryugreen** [comment]
Yeah i think exactly that - what do we need to know about it to understand the modeling. The fact sheet has all the other details about the facility/project and we don't need to duplicate that effort.

**Comment by rburghol** [comment]
Awesome. I'll do a little bit of work on this for the James write-up prior to today's meeting.

**Comment by rburghol** [comment]
@jdkleiner can you find the email where I laid out a couple of names for the attributes we want to stash for doing custom table headers and such?  I can't for the life of me find it, or maybe I stuck it somewhere in code...  I want to get it in the body of this issue so that we can insure that we are using a consistent naming convention.

**Comment by rburghol** [comment]
Error running James River:
```Quitting from lines 324-366 (VWP_CIA_Summary.Rmd) 
Error: tinyformat: Too many conversion specifiers in format string```
